CMakeLists.txt

```diff
@@ -97,6 +97,17 @@ install(DIRECTORY include/circt
   PATTERN "LICENSE.TXT"
   )
 
+install(DIRECTORY include/circt-c
+  DESTINATION include
+  COMPONENT circt-headers
+  FILES_MATCHING
+  PATTERN "*.def"
+  PATTERN "*.h"
+  PATTERN "*.inc"
+  PATTERN "*.td"
```

Comment:
There are no inc, td, or def files in the c-api.

Reply:
I copied this verbatim from LLVM. I don't see a problem leaving these in unless we explicitly want to take the stance that we are trying to build this API without these file types.